origin detection over UDP (#4532)
DataDog/documentation: content/en/agent/kubernetes/dogstatsd.md

With this, any pod running your application is able to send DogStatsD metrics via port `8125` on `$DOGSTATSD_HOST_IP`.

## Origin detection over UDP

Origin detection allows DogStatsD to detect where the container metrics come from, and tag metrics automatically. When this mode is enabled, all metrics received via UDP are tagged by the same container tags as Autodiscovery metrics.

> **Note**: as an alternative to UDP, you may also use [Unix Domain Sockets][18].

Reviewer: ```suggestion
**Note**: An alternative to UDP is [Unix Domain Sockets][18].
```
